src/EnergyPlus/HVACFourPipeBeam.hh

```diff
@@ -0,0 +1,280 @@
+#ifndef HVACFourPipeBeam_hh_INCLUDED
+#define HVACFourPipeBeam_hh_INCLUDED
+
+#include <memory>
+// ObjexxFCL Headers
+#include <ObjexxFCL/Array1.hh>
+
+// EnergyPlus Headers
+#include <AirTerminalUnit.hh>
+#include <EnergyPlus.hh>
+#include <DataGlobals.hh>
+#include <GlobalNames.hh>
+#include <PlantLocation.hh> // this is from https://github.com/NREL/EnergyPlus/pull/4945  assuming it goes in other wise maybe the same from using DataPlant
+
+namespace EnergyPlus {
+
+namespace FourPipeBeam {
+
+class HVACFourPipeBeam : public AirTerminalUnit
+{
+
+public: // Creation
+
+	// Default Constructor
+	HVACFourPipeBeam() :
```

Comment:
@EnergyArchmage You are calling the factory method in ZoneAirLoopEquipmentManager.cc, not the class constructor. I guess the question is whether we want the constructors to be public or private (if they truly can be private)? @Myoldmopar @kbenne thoughts?